src/main/java/spark/utils/GzipUtils.java

```diff
@@ -68,8 +68,10 @@ public static OutputStream checkAndWrap(HttpServletRequest httpRequest,
 
         if (acceptsGzip) {
             if (!requireWantsHeader || wantGzip) {
+            	if(spark.Spark.doGzip==true){
```

Comment:
Thanks for the feedback! (I'm new to java programming). I agree with getters and setters general - but I was trying to mimic the way Spark does things more generally. So in the server setup you do not generate an instance of the class - so I thought that a static variable would be better here. (But I'm very happy to be told I'm wrong).